source/enumerate-indexes.rst

```diff
@@ -353,13 +358,15 @@ Tests
 
   - verify that *all* index names are represented in the result
   - verify that there are no duplicate index names
-  - there are no returned indexes that do not exist
+  - verify there are no returned indexes that do not exist
 
 - Run the driver's method that returns a list of index information records, and:
 
-  - all the indexes are represented in the result
-  - the "unique" flags show up for the unique index
-  - there are no duplicates in the returned list
+  - verify all the indexes are represented in the result
+  - verify the "unique" flags show up for the unique index
+  - verify there are no duplicates in the returned list
+  - if the result consists of statically defined index models, verify that the ``ns`` fields are accurate if such a field is
```

Comment:
I think this could be rewritten as:

> if the result consists of statically defined index models that include an `ns` field, verify that its value is accurate

Reply:
done